presto-native-execution/presto_cpp/main/operators/LocalShuffle.h

```diff
@@ -77,12 +104,11 @@ class LocalShuffleWriter : public ShuffleWriter {
       uint32_t shuffleId,
       uint32_t numPartitions,
       uint64_t maxBytesPerPartition,
+      bool isSortShuffle,
```

Comment:
Let's make sure all locations are updated in terms of the naming, to be consistent